cmd/plugin/builder/template/root.go

```diff
@@ -52,6 +52,12 @@ var Makefile = Target{
 	Template: plugintemplates.Makefile,
 }
 
+// PluginToolingMK target
+var PluginToolingMK = Target{
+	Filepath: "plugin-tooling.mk",
+	Template: plugintemplates.PluginToolingMK,
+}
+
```

Comment:
Unrelated, but Line 65 below caught my eye :-)

I am not sure if there is a best way to address this.
how about remove the TODO about it, and
change 
```
Template: `* @vuil`,
```
to
```
Template: `* # edit as appropriate`,
```